layouts/partials/future.html

```diff
@@ -8,6 +8,7 @@
       {{- if ne ($.Scratch.Get "month") (dateFormat "January" .startdate ) -}}
         {{- $.Scratch.Set "month" (dateFormat "January" .startdate ) -}}
         {{- $.Scratch.Set "month-displayed" "false" -}}
+<br />
```

Comment:
Fair. I was following the usage already present in this partial, but I don't think it's perfect. It's just less annoying than it was.